src/components/App/App.js

```diff
@@ -21,6 +21,7 @@ import About from '../guidance-content/about/About';
 import logo from '../../assets/logo.png';
 import marmot from '../../assets/marmot.png';
 import styles from './App.css';
+import '!style!css!react-input-range/lib/css/index.css';
```

Comment:
It's magic syntax for the webpack CSS loader(s). I don't understand it very well, but what it does is force the styles to be loaded as global styles rather than run through the localizer that creates unique local style names extended with hashes, e.g., `myclass__a09fxbb8. 

FWIW, there are discussions [here](https://github.com/css-modules/css-modules/pull/65) and [here](https://github.com/JedWatson/react-select/issues/176).